create_release.sh

```diff
@@ -31,4 +31,4 @@ git push https://${GH_WRITE_TOKEN}@github.com/${TRAVIS_REPO_SLUG}.git ${NEW_VERS
 
 # create the release in github
 body="{\"tag_name\": \"${NEW_VERSION}\", \"name\": \"${NEW_VERSION}\"}"
-curl -H "Authorization: token ${GH_WRITE_TOKEN}" -H "Content-Type: application/json" -d "${body}" https://api.github.com/repos/DepartmentOfHealth-htbhf/htbhf-deployment-scripts/releases
+curl -H "Authorization: token ${GH_WRITE_TOKEN}" -H "Content-Type: application/json" -d "${body}" https://api.github.com/repos/${TRAVIS_REPO_SLUG}/releases
```

Comment:
These should be the same values, but is easier to maintain if the repo name changes or (as in my case), this line is copied for use elsewhere.